plan/plan.go

```diff
@@ -0,0 +1,75 @@
+package plan
+
+// DNSRecord holds information about a DNS record.
+type DNSRecord struct {
```

Comment:
Yu have struct `Plan` in file `plan.go` in package `plan`, this is what I mean. I think the naming is poor and could be improved.